README.md

```diff
@@ -9,16 +9,16 @@ It is deployed as a web application in a WebLogic Server (WLS) instance, version
 ## Downloading the release
 
 You can find all the releases on the [releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases).  
-To download the web application wls-exporter.war and put your configuration file into the WAR, download the getXXX.sh script and then run:
+To download the web application `wls-exporter.war` and put your configuration file into the WAR, download the `getXXX.sh` script and then run:
```

Comment:
Its not clear where the user should download the getXXX.sh script.  I had to go ask.  If that aspect is covered in documentation elsewhere, that's fine too.  But if not, it might help to let the user know that the getXXX.sh script is provided with each release and can be downloaded from the release page.

Reply:
It's obscure in the docs, stating only, "You can find all the releases on the [releases page]." I'll make it more clear as per your suggestion.